frontend/src/components/pages/site/OutdoorCourseUI.tsx

```diff
@@ -97,362 +97,306 @@ export const OutdoorCourseUIWithoutContext: React.FC<Props> = ({ outdoorCourseUr
-                      bbox={outdoorCourseContent.bbox}
-                      trekChildrenGeometry={[]}
-                      touristicContentPoints={outdoorCourseContent.touristicContents
-                        .filter(touristicContent => touristicContent.geometry !== null)
-                        .map(touristicContent => ({
-                          // It's ok to ignore this rule, we filtered null values 2 lines above
-                          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                          geometry: touristicContent.geometry!,
-                          pictogramUri: touristicContent.category.pictogramUri,
-                          name: touristicContent.name,
-                          id: `DETAILS-TOURISTIC_CONTENT-${touristicContent.id}`,
-                        }))}
-                      sensitiveAreas={outdoorCourseContent.sensitiveAreas
-                        .filter(sensitiveArea => sensitiveArea.geometry !== null)
-                        .map(({ geometry, color }) => ({
-                          geometry,
-                          color,
-                        }))}
-                      trekId={Number(id)}
-                      title={outdoorCourseContent.name}
-                      signage={outdoorCourseContent.signage}
-                      service={outdoorCourseContent.service?.map(service => ({
-                        location: { x: service.geometry.x, y: service.geometry.y },
-                        pictogramUri:
-                          service.type.pictogram ?? renderToStaticMarkup(<MapPin color="white" />),
-                        name: service.type.name,
-                        id: `DETAILS-SERVICE-${service.id}`,
-                      }))}
-                      infrastructure={outdoorCourseContent.infrastructure}
-                    />
-                  </div>
-                )}
+                <Footer />
               </div>
-            </div>
-
-            {isMobile && (
-              <MobileMapContainer
-                id="outdoorCourseContent_mobileMap"
-                className={`desktop:hidden fixed right-0 left-0 h-full z-map ${
-                  mobileMapState === 'HIDDEN' ? 'invisible' : 'flex'
-                }`}
-                displayState={mobileMapState}
+              <div
+                id="outdoorCourseContent_map"
+                className={cn(
+                  'fixed inset-0 z-map left-full w-full transition',
+                  'desktop:flex desktop:z-content desktop:bottom-0 desktop:fixed desktop:left-auto desktop:right-0 desktop:w-2/5 desktop:top-headerAndDetailsRecapBar',
+                  mobileMapState === 'DISPLAYED'
+                    ? '-translate-x-full desktop:translate-x-0'
+                    : 'translate-x-0',
+                )}
               >
                 <DetailsMapDynamicComponent
-                  type="MOBILE"
+                  type={isMobile ? 'MOBILE' : 'DESKTOP'}
```

Comment:
Plutôt que de faire ce 'isMobile' partout, pouquoi pas juste rajouter une prop isMobile sur `DetailsMap` ?
Instead of puting this `isMobile` everywhere, why not just add a prop `isMobile` on `DetailsMap`?

With just a ternary like this, `type={isMobile ? 'DESKTOP : 'MOBILE'}` would be valid for the linter and hard to spot for an human

Reply:
Wait, `type` in DetailsMap is only used for this? https://github.com/GeotrekCE/Geotrek-rando-v3/blob/03b6ab0d97a9f8734ba40495fc3b09beb200e7cb/frontend/src/components/Map/DetailsMap/DetailsMap.tsx#L162

Why not just a `hasZoomControl` prop then? I thought there was a whole render logic associated with the type of map. Or maybe I'm not looking in the right spot?

Reply:
Yes, you're right, that wasn't the point of this PR, however i fix that!